log/zap.go

```diff
@@ -43,8 +43,8 @@ func newZapLogger(name, env string) (Logger, error) {
 	// Create zap cores that will be populated based on env configuration
 	cores := []zapcore.Core{}
 
-	skipStdoutLogger, err := strconv.ParseBool(os.Getenv(loggerKeySkipStdoutLogger))
-	if err != nil || !skipStdoutLogger {
+	stdoutLogger, _ := strconv.ParseBool(os.Getenv(loggerKeyStdoutLogger))
+	if stdoutLogger {
```

Comment:
This will not work. This will break our entire production deployment because we depend on stdout logs in container which GKE collects and stores in Google Cloud logging.

If you want this behaviour by default, then I suggest do the following:

- Introduce a new `ZapLoggerConfig` with flags that you need struct
- Introduce `DefaultZapLoggerConfig()` which creates a default config maintaining existing version
- Introduce a new method called `InitZapLoggerWithConfig` taking `name, env, config` as arg
- Update the private method `newZapLogger` to take `config` as param
- Skip stdout based on config flags
- Store config in `zapLoggerWrapper` for future enhancement